### PR TITLE
DE54258: Throw InvalidTokenException if token claims don't contain va…

### PR DESCRIPTION
**Pull request by irinaepshteyn**
…lid 'issued at' and 'expiration' dates.

### CODE REVIEW

**Comment by irinaepshteyn** [comment]
Test PASSed:
https://predix1.jenkins.build.ge.com/job/Predix-Security/job/token-lib/job/token-lib-dev-pullrequest/125/

File: src/main/java/com/ge/predix/uaa/token/lib/FastTokenServices.java [hunk 1]
```diff
@@ -180,10 +180,16 @@ private void verifyIssuer(final String iss) {
         }
     }
 
-    private void verifyTimeWindow(final Map<String, Object> claims) {
+    protected void verifyTimeWindow(final Map<String, Object> claims) {
```
Comment (sanjeevchopra) [inline src/main/java/com/ge/predix/uaa/token/lib/FastTokenServices.java]: does this need to be 'protected'?

Comment (irinaepshteyn) [reply src/main/java/com/ge/predix/uaa/token/lib/FastTokenServices.java]: @sanjeevchopra, to expose the method to the test.  I could have it package-protected instead, I guess...

Comment (sanjeevchopra) [reply src/main/java/com/ge/predix/uaa/token/lib/FastTokenServices.java]: yes, please do

**Pull request merged**